libdnf/rpm/transaction.cpp

```diff
@@ -625,8 +625,8 @@ class Transaction::Impl {
                 cb.verify_stop(total);
                 break;
             case RPMCALLBACK_UNKNOWN:
-                std::cout << "RPMCALLBACK_UNKNOWN" << std::endl;
-                break;
+                // TODO (lukash) handle the case properly, LogicError might not be the correct way
+                throw LogicError("RPM callback unknown");
```

Comment:
I think throwing an error in this part of code is exactly what we must avoid. We must not terminate rpm transaction for any reason. We can only report it. Nothing more.

Reply:
This is a critical problem!

Reply:
The change is also hidden in commit marked as moving and cleanups, but this is a critical change.

Reply:
`std::cout` is certainly not the way to handle the case either. It's not like this is production ready anyway, at least now there's a TODO.

I'll fix it...

Reply:
I am fine with TODO.

Reply:
@jrohel I've changed the throw to a logger warning, I think that's about what we want here, so I also removed the TODO, or do we still want it there to revisit later?